### PR TITLE
[release/v2.7] Remove carriage-return characters

### PR DESCRIPTION
**Pull request by superseb**
Found this while backporting the BCI bump to `release/v2.6`, see `package/Dockerfile` example below:

```
$ cat -v package/Dockerfile 
FROM registry.suse.com/bci/bci-micro:15.4^M
^M
ARG user=adapter^M
^M
RUN echo "$user:x:1000:1000::/home/$user:/bin/bash" >> /etc/passwd && \^M
    echo "$user:x:1000:" >> /etc/group && \^M
    mkdir /home/$user && \^M
    chown -R $user:$user /home/$user^M
^M
COPY bin/csp-adapter /usr/bin/^M
^M
USER $user^M
^M
CMD ["csp-adapter"]^M
```

This PR was generated using the command `find . -type f -not -path '*/\.git/*' -print0 | xargs -0 dos2unix`.

### CODE REVIEW

**Comment by MbolotSuse** [comment]
@superseb Can you explain the impact that you are seeing from the use of these characters? If this is causing a specific issue I'll take a look at changing it, but otherwise this touches a lot of files, so I'm a bit apprehensive about pulling it in.

**Comment by superseb** [comment]
@MbolotSuse This usually causes a lot of issues as it can break parsing files. Apparently nothing has broken yet but I would prefer to get rid of them asap, the only important files touched are chart and the dockerfile, both which can be easily tested after releasing a new RC.

If you want to postpone it to a later release, I can put it in draft and just backport the BCI bump as-is to `release/v2.6`.

**Comment by MbolotSuse** [comment]
Thanks for explaining. I can give this a look, but it will take me a moment since I need to do some basic testing to make sure that this doesn't break any install/upgrade cases.

**Comment by superseb** [comment]
Okay cool, thanks. I created https://github.com/rancher/csp-adapter/pull/31 just in case for the minimal backport.

**Review by MbolotSuse — APPROVED** [review]
Basic install/uninstall/build cases lgtm.

File: .gitignore [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+/test
+/example
+/.dapper
+/.cache
+/bin
+/build
+/dist
+*.swp
+.idea
 .envrc
```
Comment (maxsokolovsky) [inline .gitignore:11]: Nit: is the addition of `.envrc` deliberate?

**Review by maxsokolovsky — APPROVED** [review]
Left a question about whether .envrc was added to .gitignore deliberately.
Aside from that, everything looks good. Tested by using Sublime Text's builtin text selection with each diff (before and after blocks). If there are character differences, then Sublime won't select the "after" block after selecting the "before" block.

File: .gitignore (hunk 1 above)
Comment (superseb) [reply .gitignore:11]: it is not an addition ;) it is the only line that doesn't have the character

**Comment by superseb** [comment]
@MbolotSuse @maxsokolovsky can we merge this?

**Pull request merged**